frontend/dockerfile/docs/reference.md

```diff
@@ -1158,20 +1158,25 @@ The latter form is required for paths containing whitespace.
 > Only octal notation is currently supported. Non-octal support is tracked in
 > [moby/buildkit#1951](https://github.com/moby/buildkit/issues/1951).
 
+> **Note**
+>
+> The `--exclude` option can be specified multiple times and cause files matching its patterns not to be copied,
+> even if the files paths match the pattern specified in `<src>`. This feature requires the build tag `dfexcludepatterns`.
```

Comment:
"This feature requires using labs channel"

cc @dvdksn to double check this before the release. Maybe we need to separate it more. I also see that `--parents` is not mentioned for `COPY` that should be similar.